### DISCUSSION TITLE
Welcome to MindNote Discussions!

### DISCUSSION THREAD

**Discussion by miscmo** [miscmo/MindNote#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by linkangsun** [comment]
1. 这个项目为什么用c++开发，是出于哪些因素的考量?
2. 这个项目的架构设计？
3. 这个项目希望在多久达到什么样的完成度？
4. 市场调研是怎样的？
